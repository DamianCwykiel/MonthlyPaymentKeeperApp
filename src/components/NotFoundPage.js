import React from 'react';
import {Link} from 'react-router-dom';

export const NotFoundPage = () => (
    <div className="content-container__edit">
        <h4 className="header__title">
             <b>404! - Page not found.</b>
        </h4>
            <Link to='/'>Home Page</Link>
    </div>
);
export default NotFoundPage;