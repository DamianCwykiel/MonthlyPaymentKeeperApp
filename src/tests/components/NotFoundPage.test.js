import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

//Adapter configuration
Enzyme.configure({ adapter: new Adapter() });

test ('shpuld render NotFoundPage correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});