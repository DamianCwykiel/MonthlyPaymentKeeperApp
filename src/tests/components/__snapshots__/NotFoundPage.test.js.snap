// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shpuld render NotFoundPage correctly 1`] = `
<div>
  <h4>
    <b>
      404! - Page not found.
    </b>
  </h4>
  <Link
    replace={false}
    to="/"
  >
    Home Page
  </Link>
</div>
`;
